static/panes/cfg-view.ts

```diff
@@ -165,7 +169,7 @@ export class Cfg extends Pane<CfgState> {
             const prevZoom = this.state.zoom;
             this.state.zoom += delta;
             if (this.state.zoom >= MINZOOM) {
-                this.graphElement.style.transform = `scale(${this.state.zoom})`;
+                this.graphElement.style.transform = `scale(${this.state.zoom})${this.extraTransforms}`;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/transform only shows this multiple transforms with spaces inbetween. To me, the specs are unclear on if no space will work in every browser. Do you have a source on this?